resources/views/admin/location-edit.phtml

```diff
@@ -91,103 +91,105 @@ use Fisharebest\Webtrees\View;
-                    $('form').trigger('reset');
-                    return false;
-                };
-                let reset = config.i18n.reset;
-                let anchor = L.DomUtil.create('a', 'leaflet-control-reset', container);
-                anchor.setAttribute('aria-label', reset);
-                anchor.href = '#';
-                anchor.title = reset;
-                anchor.role = 'button';
-                let image = L.DomUtil.create('i', 'fas fa-redo', anchor);
-                image.alt = reset;
-
-                return container;
-            }
-        });
-
-        // Geocoder (place lookup)
-        let geocoder = new L.Control.geocoder({
-            position: 'bottomleft',
-            defaultMarkGeocode: false,
-            expand: 'click',
-            showResultIcons: true,
-            query: <?= json_encode($location->locationName(), JSON_THROW_ON_ERROR) ?>,
-            placeholder: <?= json_encode(I18N::translate('Place'), JSON_THROW_ON_ERROR) ?>,
-            errorMessage: <?= json_encode(I18N::translate('Nothing found.'), JSON_THROW_ON_ERROR) ?>,
-            iconLabel: <?= json_encode(I18N::translate('Search'), JSON_THROW_ON_ERROR) ?>
-        })
-            .on('markgeocode', function (result) {
-                let coords = result.geocode.center;
-                let place = result.geocode.name.split(',', 1).toString();
-                marker.setLatLng(coords);
-                map.panTo(coords);
-                if (add_place) {
-                    $('#new_place_name').val(place);
-                }
-                $('#new_place_lati').val(Number(coords.lat).toFixed(5));
-                $('#new_place_long').val(Number(coords.lng).toFixed(5));
-            });
-
-        /**
-         * @private
-         */
-        $(function () {
-            // geocoder button tooltip
-            $('.leaflet-control-geocoder-icon')
-                .attr('title', <?= json_encode(I18N::translate('Search'), JSON_THROW_ON_ERROR) ?>);
-
-            $('.editable').on('change', function () {
-                let lat = $('#new_place_lati').val();
-                let lng = $('#new_place_long').val();
-                marker.setLatLng([lat, lng]);
-                map.panTo([lat, lng]);
-            });
-        });
-
-      const map = webtrees.buildLeafletJsMap('osm-map', config)
-            .addControl(new resetControl())
-            .addControl(geocoder)
-            .addLayer(marker)
-            .fitBounds(<?= json_encode($map_bounds, JSON_THROW_ON_ERROR) ?>, {padding: [50, 30]})
-            .on('zoomend', function () {
-              if (!map.getBounds().contains(marker.getLatLng())) {
-                map.panTo(marker.getLatLng());
-              }
-            });
-
-        return 'Leaflet map interface for webtrees-2';
-    })();
+  'use strict';
+
+  window.WT_ADMIN_MAP = (function () {
```

Comment:
Do we need this assignment (and the corresponding `return` at the bottom?

I can't see that it is used anywhere?

Reply:
Can't remember why it was originally done but it can be removed - I'll do it

Reply:
Done